ui/components/app/new-collectibles-notice/new-collectibles-notice.component.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Box from '../../ui/box';
+import Dialog from '../../ui/dialog';
+import Typography from '../../ui/typography/typography';
+import {
+  COLORS,
+  TYPOGRAPHY,
+  TEXT_ALIGN,
+  FONT_WEIGHT,
+  DISPLAY,
+} from '../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export default function NewCollectiblesNotice() {
+  const t = useI18nContext();
+
+  return (
+    <Box marginBottom={8}>
+      <Dialog type="message">
+        <Box display={DISPLAY.FLEX}>
+          <Box paddingTop={2}>
+            <i style={{ fontSize: '1rem' }} className="fa fa-info-circle" />
+          </Box>
+          <Box paddingLeft={4}>
+            <Typography
+              color={COLORS.BLACK}
+              align={TEXT_ALIGN.LEFT}
+              variant={TYPOGRAPHY.Paragraph}
+              fontWeight={FONT_WEIGHT.BOLD}
+            >
+              {t('newNFTsDetected')}
+            </Typography>
```

Comment:
This component is nice convenience function for i18n: https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/ui/i18n-value/i18n-value.component.js

`<I18nValue messageKey="dummyKey" />`